.vale/styles/RedHat/CaseSensitiveTerms.yml

```diff
@@ -53,6 +53,7 @@ swap:
   'Red\sHat\sVirtualization\sHypervisor|RHV\sHost|RHV-H': Red Hat Virtualization Host
   'RHVM|RHV-M|RHV\sManager': Red Hat Virtualization Manager
   'self-hosted\sengine\svirtual\smachine|engine\sVM': Manager virtual machine
+  "(?<!the |the Red Hat )Hybrid Cloud Console|(?<!the )Red Hat Hybrid Cloud Console|(?<!the |Red Hat |Hybrid )Cloud Console|the Cloud Console|HCC|the HCC|the RH HCC": the Red Hat Hybrid Cloud Console on first mention|the Hybrid Cloud Console
```

Comment:
```suggestion
  "(?<!the |the Red Hat )Hybrid Cloud Console|(?<!the )Red Hat Hybrid Cloud Console|(?<!the |Red Hat |Hybrid )Cloud Console|the Cloud Console|HCC|the HCC|the RH HCC": the Red Hat Hybrid Cloud Console|the Hybrid Cloud Console
```

Reply:
The reason for removing "on first mention" is that we don't want to add descriptive text into the replacement term